devices/System.Buffers.Helpers/BitConverter/LittleEndianBitConverter.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers.Binary;
+
+namespace System.Buffers.Helpers.BitConverter
+{
+    /// <summary>
+    /// The <see cref="LittleEndianBitConverter"/> class offers methods to convert to and from a byte[] in a Little Endian manner.
+    /// </summary>
+    public sealed class LittleEndianBitConverter : EndianBitConverter
```

Comment:
why do you want the class to be sealed?
I don't see any real value. If someone wants to extend it it's not going to be possible anymore.